### PR TITLE
Water ops

### PR DESCRIPTION
**Pull request by goertler**
script and plots to investigate influence of water ops for next meeting

### CODE REVIEW

**Review by ryanpeek — CHANGES_REQUESTED** [review]
@goertler I would recommend not pushing the tiff's up (in fact, I would ignore them completely using a line in your .gitignore "figures/*.tiff". They are very large (+20 MB each), and so pushing and pulling them will take a long time, and will happen every time they need updating. Better to save a version as a jpg or png in the interim, and when needed you can generate the tiff for production/manuscript locally (and not push it up).

File: investigate_water_ops.R [hunk 1]
```diff
@@ -15,8 +15,13 @@ all_flows_test$pch <- ifelse(all_flows_test$cooks > cutoff, 1, 2)
 all_flows_test$test <- all_flows_test$YOLO/all_flows_test$SAC
 mean(all_flows_test$test) # 0.02765974
 
-plot(all_flows_test$SAC, all_flows_test$YOLO, xlim = c(0,77500), ylim = c(0,19083), col = all_flows_test$col, pch = all_flows_test$pch)
+tiff(filename = "figures/sac_vs_yolo.tiff", width = 10, height = 8, units = "in", res = 300)
```
Comment (ryanpeek) [inline investigate_water_ops.R:18]: use png or pdf here... much smaller file sizes and still good resolution

File: investigate_water_ops.R [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# data from f_clean_inundation
+
```
Comment (ryanpeek) [inline investigate_water_ops.R:2]: Need a read.csv or load data of some sort...can't duplicate this code without the dataset?

**Comment by ryanpeek** [comment]
@goertler @catarfish I made a few minor changes to the code, and removed the tiffs from this pull request. Make sure to pull to your branch in your repo first (to incorporate these edits), then you can rerun and fix the bit at the beginning with the data, regenerate the plots as pngs, and then commit and push, we can merge things in pretty quickly! Let me know if you have questions.

**Comment by goertler** [comment]
Thanks Ryan, done and done (I ended up changing the inundation data file a bit so that this R script can just pull from one source).

**Pull request merged**